src/components/ProposalStatusForm.tsx

```diff
@@ -14,6 +15,8 @@ export default function ProposalStatusForm({
   proposalDetails,
   session,
 }: ProposalStatusFormProps) {
+  const [feedbackGiven, setFeedbackGiven] = useState(false)
```

Comment:
useState -> state will be lost on, e.g., page reload. use something like session storage to persist the state for a longer period. for example, using a hook like https://usehooks.com/usesessionstorage (stored for the duration of the browser session).